Sujith/JavaScript/Assignment6/assignment.js

```diff
@@ -0,0 +1,78 @@
+// Program 1
+function myfun1() {
+    let names = [];
+    for (let i = 0; i < 5; i++) {
+        names.push(prompt("Enter a name"));
+    }
+    document.getElementById('res1').innerHTML = names.toString();
```

Comment:
i=0

Reply:
It is 1 because I want to print Name 1, Name 2 ...
So I have taken i = 1 so I can use Name (i).